sdk/python/lib/test/test_next_serialize.py

```diff
@@ -246,3 +246,25 @@ def test_unsupported_sig(self):
         except  AssertionError as err:
             error = err
         self.assertIsNotNone(error)
+
+    def test_secret_push_up(self):
+        secret_value = {rpc._special_sig_key: rpc._special_secret_sig, "value": "a secret value" }
+        all_props = struct_pb2.Struct()
+        all_props["regular"] = "a normal value"
+        all_props["list"] = ["a normal value", "another value", secret_value]
+        all_props["map"] = {"regular": "a normal value", "secret": secret_value}
```

Comment:
Can this test include nested instances of these candles (lists in maps and maps in lists) for better coverage?

I mention it in part because the python changes looked at first like they could potentially lead to issues with nested secrets not being deserialized correctly anymore - though I think I’ve convinced myself that is not a concern.